### ISSUE TITLE
FileInputFieldWithList placeholder never shows

### ISSUE THREAD

**Issue by ismay** [dhis2/ui#70]
I was working on the default translations for the FileInputFields and noticed that the placeholder for FileInputFieldWithList wasn't showing. The logic that decides whether a placeholder should show is in `FileInputField.js`:

```jsx
                {!children && placeholder ? (
                    <FileListPlaceholder>
                        {translate(placeholder)}
                    </FileListPlaceholder>
                ) : (
                    children
                )}
```

The problem is that the `FileInputFieldWithList` passes this as children to the FileInputField:

```jsx
                {files.map(file => (
                    <FileListItemWithRemove
                        key={file.name}
                        label={file.name}
                        removeText={translate(removeText)}
                        onRemove={this.handleRemove}
                        file={file}
                    />
                ))}
```

The default value for files is an array, so it'll always pass at least an empty array. So the children are never empty, and the placeholder will never show up. Unless I'm overlooking something. At any rate, the FileInputFieldWithList placeholder currently isn't showing in the story it seems.

**Comment by ismay** [comment]
Fixed in f0a7b23

**Comment by ismay** [comment]
Closed with https://github.com/dhis2/ui/pull/61
